Dockerfile.buildtests

```diff
@@ -20,7 +20,7 @@ RUN ./configure LDFLAGS="-static" && make && cp -f slirp4netns /
 
 # CentOS
```

Comment:
CentOS/SUSE/Alpine stuff can be temporarily removed from Dockerfile.tests if compiling libslirp is complicated

Reply:
I get a weird failure docker/podman failure with SUSE:
```
Error: error committing container for step {Env:[ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin] Command:run Args:[zypper install -y --no-recommends autoconf automake gcc glib2-devel git make libcap-devel libseccomp-devel meson] Flags:[] Attrs:map[] Message:RUN zypper install -y --no-recommends autoconf automake gcc glib2-devel git make libcap-devel libseccomp-devel meson Original:RUN zypper install -y --no-recommends autoconf automake gcc glib2-devel git make libcap-devel libseccomp-devel meson}: error copying layers and metadata for container "b7c800f7da8a4b8a0d4e520a32c3b5dd776783fe932c826696863dad7b879b6e": Error committing the finished image: error adding layer with blob "sha256:ee62c27ee00136106d8b63be96a746231725c8596e393e793cd1b348e4c1b268": Error processing tar file(exit status 1): unexpected EOF
```
Any idea?

Reply:
Docker? Podman?

Reply:
Should work with the latest Docker ( https://get.docker.com)

Reply:
podman rootless

Reply:
Maybe better to ask in https://github.com/containers/libpod/issues ?

cc @giuseppe

Reply:
it might be fuse-overlayfs.  Could you file a bug in libpod and please specify the version of fuse-overlayfs you are using?

Reply:
I confirm it is an issue in fuse-overlayfs, I'll take care of it.

Reply:
@giuseppe Does this PR itself LGTY?

Reply:
yes, the PR LGTM